packages/devtools/lib/src/ui/plotly.dart

```diff
@@ -1,33 +1,69 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 @JS()
 library plotly;
 
 import 'package:js/js.dart';
 
-@JS()
-external void hookupPlotlyClick(String domName, Function jsFunction);
+// Button names for modeBarButtonsToRemove
+const String SEND_DATA_TO_CLOSE = 'sendDataToCloud';
+const String SELECT_2D = 'select2d';
+const String LASSO_2D = 'lasso2d';
+const String HOVER_CLOSEST_CARTESIAN = 'hoverClosestCartesian';
+const String HOVER_COMPARE_CARTESIAN = 'hoverCompareCartesian';
+const String TOGGLE_SPIKE_LINES = 'toggleSpikelines';
+
+@JS('hookupPlotlyClick')
+external void _hookupPlotlyClick(String domName, Function jsFunction);
 
 @JS()
-external void myExtendTraces(String domName, List x0, List x1, List x2, List x3, List y0, List y1, List y2, List y3, List traces);
+external void myExtendTraces(String domName, List x0, List x1, List x2, List x3,
+    List y0, List y1, List y2, List y3, List traces);
```

Comment:
trailing comma here and everywhere else in this file

Reply:
Done.